samples/net/ipv4_autoconf/sample.yaml

```diff
@@ -1,7 +1,7 @@
 common:
   harness: net
   tags: net ipv4_autoconf
-  platform_whitelist: qemu_x86 native_posix
+  platform_whitelist: qemu_x86 native_posix same_e70_xplained
```

Comment:
Do we really need this? This is only used by sanitychecker which IMHO is fine to be run in qemu and native_posix only. This way we save some time for sanity check to finish.